pallet/staking/src/lib.rs

```diff
@@ -883,42 +909,47 @@ pub mod pallet {
 					) * HALF_POWER)
 						.saturating_add(
 							Perquintill::from_rational(l.staked_kton, kton_pool.max(1))
-								* HALF_POWER,
+								* (migration_ratio * HALF_POWER),
 						) as _
 				})
 				.unwrap_or_default()
 		}
 
 		/// Distribute the session reward to staking pot and update the stakers' reward record.
-		pub fn distribute_session_reward(amount: Balance) -> Balance {
+		pub fn distribute_session_reward(amount: Balance) {
+			let (reward_to_v1, reward_to_v2) = {
+				let reward_to_ring = amount / 2;
+				let reward_to_kton = amount - reward_to_ring;
+				#[cfg(not(any(test, feature = "runtime-benchmarks")))]
+				let ratio = migration_curve_kton_reward(T::MigrationCurve::get());
+				#[cfg(any(test, feature = "runtime-benchmarks"))]
+				let ratio = Perquintill::one();
+				let reward_to_kton_v1 = ratio * reward_to_kton;
```

Comment:
I'm sorry, my math calculation was incorrect, here ratio should be x/100

Reply:
Did you mean revert to `x/100` from `x/100+x`?

Reply:
Yes, using x directly, refer my suggestion above